src/colorize.cr

```diff
@@ -167,6 +167,29 @@ module Colorize
     def back(io : IO) : Nil
       (to_i + 10).to_s io
     end
+
+    def self.from_lscolors?(color : Char) : Color?
```

Comment:
If would be awesome if you could include some docs for this method, explaining everything you explained in the description of this PR (I had no idea what `lscolors` meant).